### ISSUE TITLE
Invalid svg xml being generated.

### ISSUE THREAD

**Issue by aespitia** [ScottPlot/ScottPlot#4104]
Issue: If you try to open the resulting SVG XML, it's invalid because it's missing a "g" tag somewhere, so it will not validate because of the unclosed tag.  

ScottPlot Version: 5.0.36

Code Sample:

```cs
ScottPlot.Plot myPlot = new();

List<PieSlice> slices = new()
{
	new PieSlice() { Value = 31.6, FillColor = Colors.Red, Label = "Category 1 - 31.6%" },
	new PieSlice() { Value = 27.2, FillColor = Colors.Orange, Label = "Category 2 - 27.2%" },
	new PieSlice() { Value = 16.2, FillColor = Colors.Gold, Label = "Category 3 - 16.2%" },
	new PieSlice() { Value = 1.4, FillColor = Colors.Green, Label = "Category 4 - 1.4%" },
	new PieSlice() { Value = 3.7, FillColor = Colors.Blue, Label = "Category 5 - 3.7%" },
	new PieSlice() { Value = 14.8, FillColor = Colors.AliceBlue, Label = "Category 6 - 31.6%" },
	new PieSlice() { Value = 3.3, FillColor = Colors.Aqua, Label = "Category 7 - 3.3%" },
	new PieSlice() { Value = 1.4, FillColor = Colors.Beige, Label = "Category 8 - 1.4%" },
	new PieSlice() { Value = 99.6, FillColor = Colors.BlueViolet, Label = "Category 9 - 99.6%" },
};

var pie = myPlot.Add.Pie(slices);
myPlot.Font.Set("Arial");
pie.DonutFraction=.6;
pie.Padding = 0;
myPlot.Legend.FontSize = 10.666667f;
myPlot.HideGrid();
myPlot.Layout.Frameless();
myPlot.ShowLegend(Edge.Right);

myPlot.SaveSvg(@"c:\temp\pietest.svg",500,375);
```

![image](https://github.com/user-attachments/assets/4f40b1f4-8a8a-420a-9e55-9aa92ab9e1a2)

**Comment by aespitia** [comment]
My guess is that it has something to do with how the XML is getting rendered from the memory stream of whatever is in there at the time, but doesn't know that the G path should be closed when the XML is grabbed.

**Comment by DDiggs91** [comment]
Agreed, adding a closing </g> tag before the end of the svg removes the error
![image](https://github.com/user-attachments/assets/824bf5ce-e6ee-4f10-8312-1b9ab43b6ce6)

**Comment by DDiggs91** [comment]
Issue appears to be [here](https://github.com/ScottPlot/ScottPlot/blob/main/src/ScottPlot5/ScottPlot5/Primitives/SvgImage.cs)
`public string GetXml()
    {
        return Encoding.UTF8.GetString(Stream.ToArray()) + "</svg>";
    }`
When the SVG is generated, it doesn't have the tag closed. We could check for open tags and close them before appending </svg>

**Comment by aespitia** [comment]
I had noticed that too, so I wasn't sure if the Stream.ToArray() should be replaced with some other way of getting the XML.  I see there's this in the API, but not sure how it's used

https://learn.microsoft.com/en-us/dotnet/api/skiasharp.skxmlstreamwriter?view=skiasharp-2.88

**Comment by aespitia** [comment]
fixed in 5.0.38
